rust/kcl-lib/tests/kcl_samples/sheet-metal-bracket/artifact_commands.snap

```diff
@@ -147,11 +147,11 @@ description: Artifact commands sheet-metal-bracket.kcl
       "segment": {
         "type": "line",
         "end": {
-          "x": -1.9218,
+          "x": 0.6364,
```

Comment:
Coordinates changed on this model also.

Reply:
I typo'd the changes to its KCL, fixed now.